packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/edgeless/docs/index.ts

```diff
@@ -31,6 +32,7 @@ export async function getDocByName(name: DocName) {
     note: (await import('./note.json')).default,
     pen: (await import('./pen.json')).default,
     shape: (await import('./shape.json')).default,
+    flow: (await import('./flow.json')).default,
```

Comment:
make these parallel using Promse.all?

Reply:
It seems to be concurrent already :)

Reply:
I guess not, sequential await is loaded one after another (n + 1 will wait for n to be finished).